3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/remove_hard_coded_jurisdiction_depth.psql

```diff
@@ -0,0 +1,1201 @@
+    jsonb_build_object(
+        'type',         'Feature',
+        'id',           structure_id,
+        'geometry',     public.ST_AsGeoJSON(structure_geometry)::jsonb,
+        'properties',   to_jsonb(row) - 'structure_id' - 'structure_geometry'
+    ) AS geojson
+FROM (
+    SELECT
+        irs_structures_report.id AS id,
+        irs_structures_report.structure_id AS structure_id,
+        irs_structures_report.structure_jurisdiction_id AS structure_jurisdiction_id,
+        irs_structures_report.structure_code AS structure_code,
+        irs_structures_report.structure_name AS structure_name,
+        irs_structures_report.structure_type AS structure_type,
+        locations.geometry AS structure_geometry,
+        irs_structures_report.task_id AS task_id,
+        irs_structures_report.plan_id AS plan_id,
+        irs_structures_report.event_date AS event_date,
+        irs_structures_report.business_status AS business_status,
+        irs_structures_report.rooms_eligible AS rooms_eligible,
+        irs_structures_report.rooms_sprayed AS rooms_sprayed,
+        irs_structures_report.eligibility AS eligibility,
+        irs_structures_report.structure_sprayed AS structure_sprayed,
+        irs_structures_report.business_status AS business_status
+    FROM pending_:schema .zambia_irs_structures_report_no_geojson AS irs_structures_report
+    LEFT JOIN locations ON irs_structures_report.structure_id = locations.id
+) row;
+
+--
+-- Move views
+--
+
+ALTER MATERIALIZED VIEW deprecated_:schema .zambia_focus_area_irs RENAME TO zambia_focus_area_irs_mopup;
+ALTER MATERIALIZED VIEW deprecated_:schema .zambia_irs_jurisdictions RENAME TO zambia_irs_jurisdictions_mopup;
+
+ALTER INDEX IF EXISTS deprecated_:schema .zambia_focus_area_irs_path_idx_gin RENAME TO zambia_fcirs_mopup_path_idx_gin;
+ALTER INDEX IF EXISTS deprecated_:schema .zambia_focus_area_irs_plan_idx RENAME TO zambia_fcirs_mopup_plan_idx;
+ALTER INDEX IF EXISTS deprecated_:schema .zambia_focus_area_irs_jurisdiction_idx RENAME TO zambia_fcirs_mopup_jurisdiction_idx;
+ALTER INDEX IF EXISTS deprecated_:schema .zambia_focus_area_irs_jurisdiction_parent_idx RENAME TO zambia_fcirs_mopup_jurisdiction_parent_idx;
+ALTER INDEX IF EXISTS deprecated_:schema .zambia_focus_area_irs_idx RENAME TO zambia_fcirs_mopup_idx;
+
+ALTER INDEX IF EXISTS deprecated_:schema .zambia_irs_jurisdictions_path_idx_gin RENAME TO zambia_irs_jurisdictions_mopup_path_idx_gin;
+ALTER INDEX IF EXISTS deprecated_:schema .zambia_irs_jurisdictions_plan_idx RENAME TO zambia_irs_jurisdictions_mopup_plan_idx;
+ALTER INDEX IF EXISTS deprecated_:schema .zambia_irs_jurisdictions_jurisdiction_idx RENAME TO zambia_irs_jurisdictions_mopup_jurisdiction_idx;
+ALTER INDEX IF EXISTS deprecated_:schema .zambia_irs_jurisdictions_jurisdiction_parent_idx RENAME TO zambia_irs_jurisdictions_mopup_jurisdiction_parent_idx;
+ALTER INDEX IF EXISTS deprecated_:schema .zambia_irs_jurisdictions_idx RENAME TO zambia_irs_jurisdictions_mopup_idx;
```

Comment:
It turns out that indices must be uniquely named for materialized views so we are renaming these to avoid conflicts.